### PR TITLE
Update Brazilian Portuguese Translation

### PR DESCRIPTION
**Pull request by altiereslima**
*Fixes #*

**Describe what the proposed change does**
-
-
-

### CODE REVIEW

File: CorsixTH/Lua/languages/brazilian_portuguese.lua [hunk 1]
```diff
@@ -872,29 +949,10 @@ map_editor_window = {
...
-    [8] = "Oh céus! Näo é um lugar agradável para ir se estiver doente.",
-    [9] = "Vou ser sincero, é um hospital normal. Francamente, eu esperava mais.",
-    [10] = "Por que me zanguei? Foi pior que ir ver uma ópera de quatro horas!",
-    [11] = "Estou aborrecido pelo que vi. Chama a isto um hospital? Mais parece uma pocilga!",
-    [12] = "Estou farto de atrair a atençäo pública e de visitar buracos fedorentos como este! Eu me demito.",
-    [13] = "Que lixo. Vou tentar fechá-lo.",
-    [14] = "Eu nunca vi um hospital täo ruim. Que vergonha!",
-    [15] = "Estou chocado. Você näo pode chamar isso de hospital! Näo me espere ver novamente.",
-    }
-  }
```
Comment (lewri) [inline CorsixTH/Lua/languages/brazilian_portuguese.lua:897]: Did you mean to delete these? We still use these and I can't see it elsewhere in this file.

Comment (altiereslima) [reply CorsixTH/Lua/languages/brazilian_portuguese.lua:897]: Oh, thank you

File: CorsixTH/Lua/languages/brazilian_portuguese.lua [hunk 2]
```diff
@@ -1,8 +1,8 @@
 --[[ Copyright (c) 2010 Manuel "Roujin" Wolf
 Copyright (c) 2012 Henrique Poyatos
 Copyright (c) 2014 Leonardo Malaman (LeonardoGamer)
-Copyright (c) 2018 Altieres Lima da Silva
 Copyright (c) 2020 Henrique Poyatos
+Copyright (c) 2024 Altieres Lima da Silva
```
Comment (lewri) [inline CorsixTH/Lua/languages/brazilian_portuguese.lua:5]:
```suggestion
Copyright (c) 2018, 2024 Altieres Lima da Silva
```

**Review by lewri — APPROVED** [review]
No need to add my suggestion yourself, I can do it before merge.

**Pull request merged**